src/spec/integration_support/dummy_cpi.rb

```
@@ -0,0 +1,920 @@
+
+  def network_cids
+    Dir.glob(network_file('*')).map { |network| File.basename(network) }.shuffle
+  end
+
+  def kill_agents
+    vm_cids.each { |agent_pid| kill_process(agent_pid) }
+  end
+
+  def kill_process(agent_pid)
+    Process.kill('KILL', agent_pid.to_i)
+    # rubocop:disable HandleExceptions
```

RuboCop (1.88.0)

[warning] Unnecessary disabling of `HandleExceptions` (unknown cop).

(Lint/RedundantCopDisableDirective)